sdlib/d/rt/trampoline.d

```diff
@@ -96,7 +96,15 @@ int resolve_pthread_create(pthread_t* thread, const pthread_attr_t* attr,
 		cast(PthreadCreateType) dlsym(RTLD_NEXT, "pthread_create");
 	if (real_pthread_create is null) {
 		import core.stdc.stdlib, core.stdc.stdio;
-		printf("Failed to locate pthread_create!");
+		printf("Failed to locate pthread_create!\n");
+		exit(1);
+	}
+
+	if (cast(void*) real_pthread_create is cast(void*) &pthread_create) {
```

Comment:
Note, without the cast(void*) I could not get this to compile with sdc.

Reply:
Ok, finally getting back to this. Please add a comment as to why you cast to `void*` and there is no need ot take the address of `pthread_create`.

Reply:
Running into this a lot in the port. Not taking the address doesn't port to normal D (it's treated like a function call instead of the address), and taking the address works in SDC. Honestly, I'd suggest we get in the habit of taking the address explicitly.

Reply:
D is wrong here.

Reply:
Well, that's an opinion you can have. But D is not changing to the SDC way. It just makes things less portable, and it means SDC cannot be considered a D compiler.

However, I can change it to the SDC way.